src/tribol/interface/tribol.cpp

```diff
@@ -717,6 +723,12 @@ void registerMortarPressures( integer meshId,
                     "on mesh " << meshId << ".");
       mesh.m_isValid = false;
    }
+   else if (pressures == nullptr && mesh.m_numCells <= 0)
```

Comment:
I think this should be >= 0 also

Reply:
see comment in registerMortarGaps()

Reply:
this has been resolved per outside discussion.